### DISCUSSION TITLE
Startup Instructions

### DISCUSSION THREAD

**Discussion by ClashSAN** [sdwebui-w-horde/sd-webui-stable-horde-worker#22] [General]
hi, I don't know anything about how stable-horde works, but I'll try testing it.
I'm running with 0000.. api key, I've enabled the extension, what model should I be trying as a test? what "team" or such do I join, 
as the models need a specific name, what do I look at as the list?
currently "access denied" in my terminal. I probably need a direct video running example from yours truly if you can provide.

take your time, this is a great idea 😎

**Comment by MaikoTan** [comment]
This extension is intended to be a **worker** client for Stable Horde.

- You must register an account on it and use your API key. The default anonymous one (`0000000000`) would not work.
- The `team name` is actually the name of your worker, also I think I have renamed it to `worker name` in [this PR](https://github.com/sdwebui-w-horde/sd-webui-stable-horde-worker/pull/16). This name can be anything you like, which would be displayed in the worker list page like [here](https://aqualxx.github.io/stable-ui/workers)
- The current model name is [now automatically recognised by the extension itself](https://github.com/sdwebui-w-horde/sd-webui-stable-horde-worker/pull/14), so you should choose a valid model in webui before enabling the extension. All supported models are [here](https://raw.githubusercontent.com/Sygil-Dev/nataili-model-reference/main/db.json)
- I am working on a new UI that should provide a convenient way to use the extension, see #10 and #13 .

------

I know that the extension is still not "production ready", I would continue working on it as soon as possible. Because my families got COVID, I have no time to write code these days, sorry!

**Reply by ClashSAN** [reply]
thanks for the support, sorry to hear that. don't worry about the extensions, take all the time you need to recover

**Comment by db0** [comment]
Careful, there is both a "team name" and a "worker name". A team name is an team a worker can belong to for common stats. It serves no other purpose than noble competition.

**Reply by MaikoTan** [reply]
I see. Merci for the explanation.

As we have only implemented `/v2/generate/pop` and `/v2/generate/submit` APIs, that only support `worker name`. I think the current settings was enough for the usage for now.

Maybe we could add support for changing team settings in the future? I don't know, I am still learning Python and Gradio and SD stuffs.
